scripts/release/04-publish-release

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+if [[ "$#" -ne 1 ]]; then
+  echo "$0: VERSION"
+  exit 1
+fi
+
+VERSION=$1
+
+function check_status () {
+  out=$(mktemp /tmp/osorelease.XXXXX)
+  RES=$(gh api -X GET repos/{owner}/{repo}/actions/runs -f "q='branch:v${VERSION}'" | \
```

Comment:
looks like this requires folks to have the `gh` CLI installed? (I still use `hub` b/c I haven't bothered to update yet)

Reply:
yeah